cfgov/cal/views.py

```diff
@@ -63,11 +120,13 @@ def display(request, pdf=False):
     """
     display (potentially filtered) html view of the calendar
     """
+
     template_name='about-us/the-bureau/leadership-calendar/index.html'
-    form = CalendarFilterForm(request.GET)
+    form = CalendarPDFForm(request.GET) if pdf else CalendarFilterForm(request.GET)
     context = {'form': form}
+    form_is_vaild = form.is_valid()
 
-    if form.is_valid():
+    if form_is_vaild:
         cal_q = Q(('active', True))
         if form.cleaned_data.get('filter_calendar', None):
```

Comment:
Don't need None here.  it's the default optional for a get.

http://www.tutorialspoint.com/python/dictionary_get.htm